Configs/.local/share/bin/swwwallbash.sh

```diff
@@ -5,7 +5,7 @@
 
 export scrDir="$(dirname "$(realpath "$0")")"
 source "${scrDir}/globalcontrol.sh"
-wallbashImg="${1}"
+export wallbashImg="${1}"
```

Comment:
wait why is this required? for user specific things, the `${theme}/wall.set` should give you that.

Reply:
Is this file available during the wallbash loop? use case is for example auto converting the wallpaper to something.

Reply:
hey @kRHYME7 , i kinda updated the logic for clipboard position... works fine is my system... can you please give it a try for multi/vert monitors?

Reply:
sure no probs

Reply:
Almost there, But the boundaries did not respect if my scaling is not equal to 1. If it's 1 then all good

Reply:
I tried the clipboard no probs so far as well

Reply:
Ops wait lemme screencord this. Saw some multiple issues

Reply:
the positioning is working fine so as the scaling on my side

Reply:
> Almost there, But the boundaries did not respect if my scaling is not equal to 1. If it's 1 then all good

oh noooo.... what was i thinking... completely forgot to factor in scaling :)
fixing it now!

Reply:
Not much and if this is working at normal circumstances then it should be fine IG. 

Becomes too wide for my 3000*2000 monitor (scaled to 2)
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/073e4fbc-4bc4-417f-9703-2e560d36ce0d)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/0e5904fc-965d-4ffb-9c2b-c3ff0b062f66)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/64ad7aeb-dde0-48d5-bdd3-e74d73158ada)

Monitors
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/ddd8a0ee-b91d-4989-973f-941544a5d418)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/0ff020f8-9136-4648-b65e-7a688d710405)


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/739aeb36-7dde-4ec8-89ba-f7dae02db5a1)

Does not handle if monitors are mapped oddly dynamically
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/c3bc3f65-4590-43c8-953d-776e9e8bf61e)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6936462c-44bf-48f5-a712-f7c61528fc70)

Reply:
> Not much and if this is working at normal circumstances then it should be fine IG.
> 
> Becomes too wide for my 3000*2000 monitor (scaled to 2)
> 

Fixed now, please try the latest commit... it should factor in scaling now...
regarding width, i did increase it to increase the field size.

Reply:
Still got this 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/61b7f293-5e92-437f-844e-3a3e7fa1210e)

Also this 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/27719fec-0c7a-4908-ba6d-3ef4cd08aa7f)

But maybe this is an odd setup?  HAHA